test/functional/LiquidSpec.hs

```diff
@@ -0,0 +1,112 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module LiquidSpec where
+
+import           Control.Lens hiding (List)
+import           Control.Monad.IO.Class
+import           Data.Aeson
+import qualified Data.Text as T
+import           Language.Haskell.LSP.Test hiding (message)
+-- import           Language.Haskell.LSP       as LSP
+import           Language.Haskell.LSP.Types as LSP hiding (contents, error )
+import           Haskell.Ide.Engine.LSP.Config
+import           Test.Hspec
+import           TestUtils
+import           Utils
+
+-- ---------------------------------------------------------------------
+
+spec :: Spec
```

Comment:
Can this be guarded by LH being installed? That or we need to also install it in travis